artemis/config.py

```diff
@@ -154,6 +187,10 @@ class Config:
     # Shodan API key so that Shodan vulnerabilities will be displayed in Artemis
     SHODAN_API_KEY = decouple.config("SHODAN_API_KEY", default="")
 
+    # == vcs reporter settings (artemis/reporting/modules/vcs/reporter.py)
+    # Maximum size of the VCS (e.g. SVN) db file
+    VCS_MAX_DB_SIZE_BYTES = decouple.config("VCS_MAX_DB_SIZE_BYTES", default=1024 * 1024 * 5)
```

Comment:
`VCS_MAX_DB_SIZE`, leave info about `BYTES` for docs :)

Reply:
I am against this - using unit in variable name has helped me countless times - e.g. in .env one can easily sanity-check whether the value is correct.

Reply:
Fine, this is not too long.